source/developer-guides/debug.rst

```diff
@@ -0,0 +1,197 @@
+.. _debug:
+
+Source Level Debugging
+--------------
+
+This explains how to perform basic debugging operation with Intel UDK Debugger on SBL over serial port. (used Apollo Lake (ABL) as example)
+
+
+Prerequisite
+^^^^^^^^^^^^^^
```

Comment:
* The hyperlinks in this section are not used properly. There are many examples in up2.rst. Please use it as reference to update this section.
* WinDBG requires a specific version. State we have tested with version: https://github.com/pyrasis/windowsprojectbook/blob/master/Applications/Debugging%20Tools%20for%20Windows/dbg_x86_6.11.1.404.msi